stagemonitor-requestmonitor/src/main/java/org/stagemonitor/requestmonitor/MonitoredRequest.java

```diff
@@ -34,6 +36,8 @@
 	 */
 	Object execute() throws Exception;
```

Comment:
I'd remove the method in MonitoredRequest but leave `monitor` in `RequestMonitor`, which should be deprecated. In the future, all calls whether async or not should call `monitorAsync`. So we should reconsider the name to communicate that this method is intended to be used for both async and sync calls.